From c3f14772cdcd8c477a3af05037dc262da8ac6910 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 04 Jul 2023 17:03:47 +0300
Subject: [PATCH] [SW-84954] gaudi2: remove soft reset registers

The soft reset has been transitioned to CPUCP packet and is no longer
sent through a direct register write.
Remove the gic_host_soft_rst_irq and cpu_rst_status field initialization
in dynamic regs structure passed to LKD.

Change-Id: I35d7cdc96bfd6f845555ab8060632fc1e0a84359
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index 0ce1ebb..aedaea9 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -192,8 +192,6 @@
 	regs->gic_host_pi_upd_irq = mmGIC_HOST_PI_UPD_IRQ_POLL_REG;
 	regs->gic_host_halt_irq = mmGIC_HOST_HALT_IRQ_POLL_REG;
 	regs->gic_host_ints_irq  = mmGIC_HOST_INTS_IRQ_POLL_REG;
-	regs->gic_host_soft_rst_irq = mmGIC_HOST_SOFT_RST_IRQ_POLL_REG;
-	regs->cpu_rst_status = mmCPU_RST_STATUS_TO_HOST;
 	regs->eng_arc_irq_ctrl = mmENGINE_ARC_IRQ_CTRL_POLL_REG;
 }
 
